spring-cloud-dataflow-tasklauncher/spring-cloud-dataflow-tasklauncher-sink/src/test/java/org/springframework/cloud/dataflow/tasklauncher/sink/TaskLauncherSinkTests.java

```diff
@@ -66,244 +56,140 @@
 /**
  * @author David Turanski
  **/
+@SpringBootTest(classes = { TaskLauncherSinkTests.TestConfig.class },
```

Comment:
This is almost impossible to understand what has changed. I will assume it has the same or better coverage than before.